tests/unit/processor/amides/test_amides.py

```diff
@@ -38,9 +38,10 @@ def test_process_event_malicious_process_command_line(self):
         self.object.process(document)
         assert self.object.metrics.number_of_processed_events == 1
 
-        rule_attributions = document.get("rule_attributions", None)
-        assert rule_attributions
-        assert len(rule_attributions) == 10
+        result = document.get("amides")
+        assert result
+        assert result["confidence"] >= 0.32
```

Comment:
Just to understand it: Where does the `0.32` come from? Can't we fix it to the actual value `0.757`, maybe when setting the random seed? Or is it just to test a lower bounds and the actual confidence is not important.

Reply:
ah okay, got it. Thanks. Maybe you can replace it then with the variable `self.CONFIG.get("decision_threshold")`, that way we have one magic number less and it's a bit easier to understand.